evidence/xlc/2024-06-11.md

```diff
@@ -0,0 +1,40 @@
+# Retention at Rank 4
+
+|                  |                                 |
+| ---------------- | ------------------------------- |
+| **Report Date**  | Date of submission (2024/06/11) |
+| **Submitted by** | Bryan Chen                      |
+
+
+## Member details
+
+- Matrix username: @xlchen:matrix.org
+- Polkadot address: 14DsLzVyTUTDMm2eP3czwPbH53KgqnQRp3CJJZS9GR7yxGDP
+- Current rank: 4
+- Date of initial induction: NA
```

Comment:
Bryan was part of the Technical Fellowship [seeding](https://github.com/polkadot-fellows/seeding).
So, the Date of initial induction should read "seeding".